python/res/job_queue/forward_model_status.py

```diff
@@ -51,6 +52,7 @@ def __init__(self, name,
         self.error = error
         self.std_out_file = std_out_file
         self.std_err_file = std_err_file
+        self.max_memory_usage = max_memory_usage
```

Comment:
Should we add inject `reported` or similar into the name to stress that this is not a constraint on the memory usage?

Reply:
:+1: